test/test/otlp/OtlpTests.java

```diff
@@ -123,7 +131,7 @@ private static Profile getFirstProfile(ProfilesData profilesData) {
         return scopeProfiles.getProfiles(0);
     }
 
-    private static Optional<AnyValue> getAttribute(Sample sample, ProfilesDictionary dictionary, String name) {
+    public static Optional<AnyValue> getAttribute(Sample sample, ProfilesDictionary dictionary, String name) {
```

Comment:
Package-private is enough:
```suggestion
    static Optional<AnyValue> getAttribute(Sample sample, ProfilesDictionary dictionary, String name) {
```